File: 1833-FindTheHighestAltitude/1833-FindTheHighestAltitude.js
// Last updated: 10/03/2026, 10:03:02
/**
 * @param {number[]} gain
 * @return {number}
 */
var largestAltitude = function(gain) {
    let current=0;
    let maxAltitude=0;
    for(let i=0;i<gain.length;i++){
        current += gain[i];             
        maxAltitude = Math.max(maxAltitude, current); 
    }
    return maxAltitude;

};

File: 1833-FindTheHighestAltitude/1833-FindTheHighestAltitude.cpp
// Last updated: 03/08/2026, 10:08:49
class Solution {
public:
    int largestAltitude(vector<int>& gain) {

        int current = 0;
        int maxAltitude = 0;

        for (int i = 0; i < gain.size(); i++) {

            current += gain[i];

            maxAltitude = max(maxAltitude, current);
        }

        return maxAltitude;
    }
};